core/src/main/java/org/mskcc/cbio/oncokb/util/IndicatorUtils.java

```diff
@@ -198,7 +198,8 @@ public static IndicatorQueryResp processQuery(Query query, String geneStatus,
 
             List<Alteration> nonVUSRelevantAlts = AlterationUtils.excludeVUS(relevantAlterations);
             Map<String, LevelOfEvidence> highestLevels = new HashMap<>();
-            List<TumorType> oncoTreeTypes = new ArrayList<>();
+            List<TumorType> relevantUpwardTumorTypes = new ArrayList<>();
```

Comment:
Could you give a few comments to explain the `relevantUpwardTumorTypes` and `relevantDownwardTumorTypes`? Thanks

Reply:
upward/downward is the relevant concept to oncotree. Getting all parent nodes with same main type in oncotree is upward. Getting all child nodes with same main type in oncotree is downward.